packages/operator/pkg/webserver/routes/v1/connection/connection_routes.go

```diff
@@ -189,6 +191,8 @@ func (cc *controller) createConnection(c *gin.Context) {
 		return
 	}
 
+	conn.Status.CreatedAt = &metav1.Time{Time: time.Now()}
```

Comment:
I thought that we will fill both CreatedAt and UpdatedAt fields during creation of entity. What do you think about it?

Reply:
what advantages a user will get in that case? Or you think that this logic of **updated field** is just more straightforward?

Reply:
added